test/phpunit.xml

```diff
@@ -2,6 +2,7 @@
 <phpunit
         bootstrap="bootstrap.php"
         verbose="true"
+        testdox="true"
```

Comment:
What is its purpose here? It was not found to be used. The rest of the code I think is fine, except for this little problem.